zulipterminal/ui_tools/views.py

```diff
@@ -1112,6 +1119,43 @@ def toggle_mute_status(self, button: Any, new_state: bool) -> None:
     def toggle_pinned_status(self, button: Any, new_state: bool) -> None:
         self.controller.model.toggle_stream_pinned_status(self.stream_id)
 
+    def keypress(self, size: urwid_Size, key: str) -> str:
+        if is_command_key('STREAM_MEMBERS', key):
+            self.controller.show_stream_members(stream_id=self.stream_id)
+        return super().keypress(size, key)
+
+
+class StreamMembersView(PopUpView):
+    def __init__(self, controller: Any, stream_id: int) -> None:
+        self.stream_id = stream_id
+        self.controller = controller
+        model = controller.model
+
+        user_ids = model.get_other_subscribers_in_stream(stream_id=stream_id)
+        user_names = [model.user_name_from_id(id) for id in user_ids]
+        sorted_user_names = sorted(user_names)
+        # sorted_user_names.insert(0, model.user_full_name)
```

Comment:
This commented code should be in this commit for showing members, right?

We should sort this in a similar way as we do the presence list (though probably not per active status). Note that showing stream members in this way might just be a temporary feature until we add filtering of the user list.